pkg/package-server/provider/inmem.go

```diff
@@ -247,13 +246,13 @@ func (m *InMemoryProvider) ListPackageManifests(namespace string) (*packagev1alp
 	return manifestList, nil
 }
 
-// GetPackageManifest implements PackageManifestProvider.GetPackageManifest(...)
 func (m *InMemoryProvider) GetPackageManifest(namespace, name string) (*packagev1alpha1.PackageManifest, error) {
 	m.mu.RLock()
 	defer m.mu.RUnlock()
 
 	var manifest packagev1alpha1.PackageManifest
 	for key, pm := range m.manifests {
+		// FIXME(alecmerdler): Needs to work with "all-namespaces"
```

Comment:
In the "all-namespaces" case would you return the first `PackageManifest` matching `name`?

Reply:
Ah nevermind, the Kubernetes API does not support getting a single item from `--all-namespaces`.
```shell
$ kubectl get pods --all-namespaces mongodb-2-6k7rg
error: a resource cannot be retrieved by name across all namespaces
```